Woof/Woof/View/ReusableViews/FiveStarRatingView.swift

```diff
@@ -9,18 +9,19 @@ struct FiveStarRatingView: View {
 
     // MARK: Internal interface
 
-    /// The rating in the range from 0 to 5 stars.
-    /// The rating equals zero, if it's not specified.
-    let rating: Int
+    /// A 5-star rating system, where 5 stars represent the highest rating
+    /// and 1 star represents the lowest rating.
```

Comment:
```suggestion
    /// A 5-Star rating.
```

All the details I can see in the `StarRating`'s documentation , you don't have to repeat them.

Reply:
Fixed.